vmdb/app/models/miq_provision/options_helper.rb

```diff
@@ -26,7 +26,7 @@ def get_source
     ems = source.ext_management_system
     raise MiqException::MiqProvisionError, "#{source.class.name} [#{source.name}] is not attached to a Management System" if ems.nil?
     raise MiqException::MiqProvisionError, "#{source.class.name} [#{source.name}] is attached to <#{ems.class.name}: #{ems.name}> that does not support Provisioning" unless MiqProvision::SUPPORTED_EMS_CLASSES.include?(ems.class.name)
-    raise MiqException::MiqProvisionError, "#{source.class.name} [#{source.name}] is attached to <#{ems.class.name}: #{ems.name}> with invalid credentials" if ems.authentication_invalid?
+    raise MiqException::MiqProvisionError, "#{source.class.name} [#{source.name}] is attached to <#{ems.class.name}: #{ems.name}> with invalid credentials" if ems.missing_credentials?
```

Comment:
I agree @blomquisg.

Waiting on @gmcculloug... I'm wondering if it makes sense to only check for the userid.  Bad password, unreachable provider or something else that authentication_check would detect are all things that this code won't protect against.